deploy/maistra-operator.yaml

```diff
@@ -4159,6 +4159,24 @@ spec:
                           type: string
                         prometheus:
                           type: object
+                        zipkin:
+                          properties:
+                            enable64bitTraceId:
+                              type: boolean
+                            maxTagLength:
+                              format: int64
+                              type: integer
+                            port:
+                              format: int64
+                              type: integer
+                            service:
+                              type: string
+                          required:
+                          - enable64bitTraceId
+                          - maxTagLength
```

Comment:
These two should be optional.

Reply:
Sorry for missing testing on an OCP. I am reverting the reflection change. The error above was missing a de-reference.
It should be `valueOfExt := reflect.ValueOf(*ext)`  However, I am getting other counter validation errors. 
I will create an integration test file in a MTT PR and then link it here.

The required field mistake was resolved in the last commit. I missed json `omitempty` in the new fields. Thanks.